src/Symfony/Component/Security/Core/Authorization/AccessDecisionManager.php

```diff
@@ -180,7 +174,7 @@ private function decideConsensus(TokenInterface $token, array $attributes, $obje
             return false;
         }
 
-        if ($grant == $deny && $grant != 0) {
+        if ($grant > 0) {
```

Comment:
This is wrong, you still need to check if there is the same number of grant and denial